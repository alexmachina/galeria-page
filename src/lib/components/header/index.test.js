import React from "react";
import renderer from "react-test-renderer";
import Header from "./";

describe("Components", () => {
  describe("Header", () => {
    it("Renders", () => {
      const tree = renderer.create(<Header />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
