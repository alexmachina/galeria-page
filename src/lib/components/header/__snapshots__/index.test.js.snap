// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header Renders 1`] = `
<div
  className="root-0-2-1"
>
  <div
    className="redBigBorder-0-2-2"
  />
  <div
    className="brand-0-2-3"
  >
    <img
      alt="Logo Galeria Pagé"
      className="logo-0-2-5 logo-0-2-4"
      src={
        Object {
          "process": [Function],
        }
      }
    />
    <h1
      className="root-0-2-6"
    >
      GaleriaPagé
    </h1>
  </div>
</div>
`;
